lib/rubocop/cop/lint/literal_in_interpolation.rb

```diff
@@ -29,7 +28,9 @@ def on_dstr(node)
         private
 
         def special_string?(node)
-          node.type == :str && !node.loc.respond_to?(:begin)
+          # handle strings like __FILE__
+          (node.type == :str && !node.loc.respond_to?(:begin)) ||
+            node.loc.expression.is?('__LINE__')
```

Comment:
I interpreted `special_string?` as the handler for special keywords (`__FILE__`, `__LINE__`, `__method__`, ...). 

My thoughts on cleanup for this would be refactor the naming of the method to make more sense, or create a new method `special_int?` to handle `__LINE__`.

What do you think?